p/pandas/build_info.json

```diff
@@ -10,6 +10,9 @@
   "validate_build_script": true,
   "use_non_root_user": false,
   "wheel_build": true,
+  "v2.2.3": {
```

Comment:
@shivansh-ibm  can you add * at end of json file and add pandas_ubi_9.3.sh  script ?

and also update the maintainer name